tokio/src/runtime/io/mod.rs

```diff
@@ -86,6 +89,7 @@ enum Tick {
 // TODO: Don't use a fake token. Instead, reserve a slot entry for the wakeup
 // token.
 const TOKEN_WAKEUP: mio::Token = mio::Token(1 << 31);
+const TOKEN_SIGNAL: mio::Token = mio::Token(1 + (1 << 31));
```

Comment:
This is in the reserved space starting at `1 << 31`.

I expect a later I/O driver refactor will change this.